tests/utils/test_password_manager.py

```diff
@@ -231,3 +232,30 @@ def test_get_http_auth_from_environment_variables(
 
     assert auth["username"] == "bar"
     assert auth["password"] == "baz"
+
+
+def test_get_pypi_token_with_env_var_positive(
+    config: Config, with_simple_keyring: None, dummy_keyring: DummyBackend
+):
+    sample_token = "sampletoken-1234"
+    repo_name = "foo"
+    manager = PasswordManager(config)
+
+    with mock.patch.dict(
+        os.environ,
+        {f"POETRY-PYPI-TOKEN-{repo_name}".upper().replace("-", "_"): sample_token},
```

Comment:
I'd rather write this as a static string: 
```suggestion
        {f"POETRY_PYPI_TOKEN_{repo_name.upper()}": sample_token},
```